### ISSUE TITLE
Does not compile on Windows

### ISSUE THREAD

**Issue by majkcramer** [polydraw/polydraw#2]
Polydraw does not compile on Windows, because current libc (Version 0.2) has some breaking changes. 

"Bindings to Windows functions not found in the CRT (e.g. those found in kernel32) were all removed. Many Windows types (like DWORD) were also removed." (Source: https://users.rust-lang.org/t/the-libc-crate-is-now-at-v0-2-0/3509)

Short Term Solution: Declare Dependency libc = "0.1" (works, tested on my w64 dev env)

Long Term Solution: "These are currently best found in the winapi10 crate for type definitions and the various *-sys crates for function bindings (e.g. kernel32-sys4)."

BTW: Interesting Project

**Comment by majorz** [comment]
Thanks for the interest in this early work-in-progress library. It is still some initial code and proof of concept examples, but things are building up little by little. Soon after we launch the website, I will post various interesting information.

The current approach with the external libraries is maintaining my own internal bindings. Initially I tried using the usual dependencies for each platform, but I found different reasons to maintain those myself. A) Initial compilation times. B) Consistency in style and consistency in higher level bindings. C) The process of writing the bindings was a great learning experience on the internals of the currently supported systems.

So I think I will define DWORD and the likes internally and keep using libc as the only dependency. I will definitely go and read what they did for the 0.2 version.
